cmd/traffic_top/Makefile.am

```diff
@@ -17,13 +17,13 @@
 
 # We only build traffic_top if libcurses and libcurl are available.
 if BUILD_TRAFFIC_TOP
-
 AM_CPPFLAGS = \
   $(iocore_include_dirs) \
   -I$(abs_top_srcdir)/lib \
   -I$(abs_top_srcdir)/mgmt/api/include \
   @CURL_CFLAGS@ \
   @CURSES_CFLAGS@
+endif
```

Comment:
I assume this change will cause build failure if curses isn't installed, because bin_PROGRAMS = traffic_top etc. are outside the BUILD_TRAFFIC_TOP conditional? If so, I think you can drop this file from the PR (git checkout master cmd/traffic_top/Makefile.am).